docs/modules/ROOT/pages/ios_accounts.adoc

```diff
@@ -18,81 +24,92 @@ These are:
 
 === Basic Authentication
 
-* Login Credentials are stored on the device (secure in system keychain).
+* Login Credentials are stored on the device (Secured in the iOS system keychain).
```

Comment:
* Login Credentials are stored on the device (Secured in the iOS system keychain).
*